crates/starknet-devnet/tests/test_dump_and_load.rs

```diff
@@ -400,4 +401,51 @@ mod dump_and_load_tests {
             panic!("Could not unpack the transaction from {loaded_transaction:?}");
         }
     }
+
+    #[tokio::test]
+    async fn set_time_with_later_block_generation_dump_and_load() {
+        let dump_file = UniqueAutoDeletableFile::new("dump_set_time");
+        let devnet_dump = BackgroundDevnet::spawn_with_additional_args(&[
+            "--dump-path",
+            &dump_file.path,
+            "--dump-on",
+            "exit",
+        ])
+        .await
+        .expect("Could not start Devnet");
+
+        // set time in past without block generation
+        let past_time = 1;
+        let set_time_body =
+            Body::from(json!({ "time": past_time, "generate_block": false }).to_string());
+        devnet_dump.post_json("/set_time".into(), set_time_body).await.unwrap();
+
+        // wait 1 second
+        thread::sleep(time::Duration::from_secs(1));
+
+        // create block
+        devnet_dump
+            .post_json("/create_block".into(), Body::from(json!({}).to_string()))
+            .await
+            .unwrap();
+        let _ = &devnet_dump
+            .send_custom_rpc("starknet_getBlockWithTxHashes", json!({ "block_id": "latest" }))
+            .await["result"];
```

Comment:
```suggestion
        devnet_dump
            .send_custom_rpc("starknet_getBlockWithTxHashes", json!({ "block_id": "latest" }));
```

Reply:
I tried that already but clippy has a problem with that "error: unused borrow that must be used"

Reply:
I tried this locally and it worked for me.

Reply:
Notice that I removed the `["result"]` part

Reply:
aa ok your version is without await

Reply:
Hm, didn't realize that

Reply:
I think the `await` is needed and it should work without `["result"]`

Reply:
Fixed in baf2e96620263f15ae67d470a056542d656648af